src/btree/bt_debug.c

```diff
@@ -962,8 +980,9 @@ __wt_debug_tree_all(void *session_arg, WT_BTREE *btree, WT_REF *ref, const char
         btree = S2BT(session);
 
     WT_WITH_BTREE(session, btree,
-      ret = __debug_tree(
-        session, ref, ofile, WT_DEBUG_TREE_LEAF | WT_DEBUG_TREE_WALK | WT_DEBUG_UNREDACT));
+      ret = __debug_tree(session, ref, ofile,
+        WT_DEBUG_TREE_LEAF | WT_DEBUG_TREE_WALK | WT_DEBUG_UNREDACT_ALL | WT_DEBUG_UNREDACT_KEYS |
```

Comment:
Shouldn't we only be passing WT_DEBUG_UNREDACT_ALL here (and not WT_DEBUG_UNREDACT_KEYS and WT_DEBUG_UNREDACT_VALUES)?

Reply:
fixed all comments from above https://github.com/wiredtiger/wiredtiger/pull/10094/commits/634b9aaa3d738e4e15d087f602f8268dc897f2b9 and https://github.com/wiredtiger/wiredtiger/pull/10094/commits/aaaa00644c9d04617b0c36a5a21caa8a333a77d8